sql-prepare-plan-cache.md

```diff
@@ -5,12 +5,14 @@ summary: Learn about SQL Prepare Execution Plan Cache in TiDB.
 
 # SQL Prepare Execution Plan Cache
 
-In the current development release of TiDB, the execution plan of prepared statements is cached by default. This includes both forms of prepared statements:
+TiDB supports execution plan caching for `Prepare` / `Execute` queries.
 
-- Using the `COM_STMT_PREPARE` and `COM_STMT_EXECUTE` protocol features.
-- Using the SQL statements `PREPARE` and `EXECUTE`.
+There are two forms of `Prepare` / `Execute` queries:
 
-The TiDB optimizer handles these two types of queries in the same way: when preparing, the parameterized query is parsed into an AST (Abstract Syntax Tree) and cached; in later execution, the execution plan is generated based on the stored AST and specific parameter values.
+- In the binary communication protocol, use `COM_STMT_PREPARE` and `COM_STMT_EXECUTE` to execute general parameterized SQL queries;
+- In the text communication protocol, use `COM_QUERY` to execute `Prepare` and `Execution` SQL queries.
+
+The optimizer handles these two types of queries in the same way: when preparing, the parameterized query is parsed into an AST (Abstract Syntax Tree) and cached; in later execution, the execution plan is generated based on the stored AST and specific parameter values.
```

Comment:
I think it would be nice to keep this text (it reads a bit clearer). But the rest LGTM

Reply:
Updated, PTAL